### PR TITLE
fix: App bar fixes

### PR DESCRIPTION
**Pull request by teolemon**
### What
- fix: App bar fixes

### CODE REVIEW

**Comment by teolemon** [comment]
This won't work, the text is white on beige background @alexgarel

File: taxonomy-editor-frontend/src/components/ResponsiveAppBar.tsx [hunk 1]
```diff
@@ -126,7 +126,7 @@ const ResponsiveAppBar = ({ displayedPages }: ResponsiveAppBarProps) => {
             >
               <MuiLink
                 sx={{ mr: 2, display: "flex", alignSelf: "center" }}
-                href="https://world.openfoodfacts.org/"
+                href="https://ui.taxonomy.openfoodfacts.net/"
                 target="_blank"
               >
```
Comment (eric-nguyen-cs) [inline taxonomy-editor-frontend/src/components/ResponsiveAppBar.tsx:131]: This should probably be a `react-router-dom` `Link` to the `/` path (you can look at how it is done below with the `Taxonomy Editor` text), as the link we want the logo to poin to depends on where the app is deployed

**Comment by teolemon** [comment]
@eric-nguyen-cs @alexgarel I made the text black. The routing stuff is currently beyond my feeble React knowledge 😅

**Comment by teolemon** [comment]
#22 30.11 src/components/ResponsiveAppBar.tsx
#22 30.11 Syntax error: ',' expected (34:54)

File: taxonomy-editor-frontend/src/components/ResponsiveAppBar.tsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ const ResponsiveAppBar = ({ displayedPages }: ResponsiveAppBarProps) => {
   };
 
   return (
-    <AppBar position="sticky" sx={{ background: "#ff8714" }}>
+    <AppBar position="sticky" sx={{ textColor: "#000" background: "#f2e9e4" }}>
```
Comment (teolemon) [inline taxonomy-editor-frontend/src/components/ResponsiveAppBar.tsx:34]:
```suggestion
    <AppBar position="sticky" sx={{ textColor: "#000", background: "#f2e9e4" }}>
```

**Pull request merged**